src/pages/p2p/components/_dp2p.tsx

```diff
@@ -1,12 +1,29 @@
 import React from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
 import { graphql, useStaticQuery } from 'gatsby'
 import device from 'themes/device'
 import { Container, SectionContainer, Show } from 'components/containers'
 import { Header, Text, QueryImage } from 'components/elements'
 import { localize } from 'components/localization'
 
+type DP2PProps = {
+    P2P: P2PProps[]
+    reverse: boolean
+}
+
+type P2PProps = {
+    title: React.ReactElement
+    subtitle: React.ReactElement
+    subtitle_mobile: React.ReactElement
```

Comment:
Import `ReactElement` 

```suggestion
    title: ReactElement
    subtitle: ReactElement
    subtitle_mobile: ReactElement
```